api/webhooks/webhooks.py

```diff
@@ -119,12 +141,14 @@ def _call_webhook_email_on_error(
         send_failure_email(webhook, data, webhook_type, res.status_code)
 
 
-def _call_webhooks(webhooks, data, event_type, webhook_type):
-    webhook_data = {"event_type": event_type.value, "data": data}
+def _call_webhooks(webhooks_id, data, event_type, webhook_type):
```

Comment:
Can we add some typehinting here? Appreciate that it didn't exist before, but we're trying to slowly improve the type hinting across the code base.

Reply:
Also the argument name should be `webhook_ids`, not `webhooks_id`.

Reply:
Done, thanks!